apps/client/src/shared/api/config/instance.ts

```diff
@@ -0,0 +1,8 @@
+import ky from 'ky';
```

Comment:
기본 ky 라이브러리는 ESM 기반이라 CommonJS 방식을 지원하지 않고 있어요. 이 부분을 개선한게 `@toss/ky` 라이브러리인데 

pnpm remove ky 로 ky 의존성을 제거하고 pnpm add @toss/ky 로 설치해주시면 좋을 것 같아요~ 

그리고 이 부분을
```suggestion
import ky from '@toss/ky';
```

이렇게 수정해주시면 됩니다!